### PR TITLE
Typescript test config added

### PR DESCRIPTION
**Pull request by MrM0ra**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+              options: {
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
+
```
Comment (garzuzo) [inline webpack.config.js]: add new line at the end of the file

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
+  
+
```
Comment (garzuzo) [inline webpack.config.js]: I mean, only put one empty line at the end of the file. Just one.

**Comment by garzuzo** [comment]
great, please continue until Step 7.

**Review by holgiosalos — APPROVED** [review]

**Review by holgiosalos — APPROVED** [review]
@MrM0ra you can squash and merge this PR, as it is already approved

**Comment by MrM0ra** [comment]
Unknown error solved: global variables for cypress environment enabled

**Pull request merged**